Assets/Scripts/Game/GameManager.cs

```diff
@@ -11,22 +11,24 @@ namespace Game
 {
     public class GameManager : MonoBehaviour
```

Comment:
Should we call it WallDangerManager since it's the major responsibility of the script? And create a separate instance for OilSpawner?

Reply:
Ok, make sense